### ISSUE TITLE
packages role fails on mgmt node on Google

### ISSUE THREAD

**Issue by christopheredsall** [ACRC/slurm-ansible-playbook#58]
On branch master, up to date to commit ACRC/citc-terraform@e4728867f357863c13ebbf36102dc3a49f457ed9 , after building the cluster and logging in as a non-privileged user, there are no compilers available:

```ShellSession
[chris@mgmt ~]$ gcc
-bash: gcc: command not found
```

It appears that none of the packages from the list have been installed  https://github.com/ACRC/slurm-ansible-playbook/blob/cdf0db949e3eeece03cc65b256e5db7d3b33708a/group_vars/management.yml#L15-L24

```ShellSession
[provisioner@mgmt ~]$ rpm -ql xorg-x11-xauth mpich-3.2-devel openmpi-devel openmpi3-devel devtoolset-8 python3-devel sos
package xorg-x11-xauth is not installed
package mpich-3.2-devel is not installed
package openmpi-devel is not installed
package openmpi3-devel is not installed
package devtoolset-8 is not installed
package python3-devel is not installed
package sos is not installed
```

Looking at `/root/ansible-pull.log` it seems that the missing `devtoolset-8` is the cause

```
[ ... ]
TASK [packages : yum install packages] *****************************************
Monday 30 September 2019  11:12:22 +0000 (0:00:00.318)       0:02:53.159 ****** 
fatal: [mgmt.c.citc-test-ce.internal]: FAILED! => changed=false 
  msg: No package matching 'devtoolset-8' found available, installed or updated
  rc: 126
  results:
  - No package matching 'devtoolset-8' found available, installed or updated
[ ... ]

PLAY RECAP *********************************************************************
mgmt.c.citc-test-ce.internal : ok=100  changed=91   unreachable=0    failed=1    skipped=8    rescued=0    ignored=1   
```

**Comment by christopheredsall** [comment]
## Workaround

Manually install the repo where software collections can be found and rerun the ansible:

```ShellSession
[provisioner@mgmt ~]$ sudo yum install centos-release-scl
[provisioner@mgmt ~]$ sudo -i
[root@mgmt ~]# ansible-playbook --start-at-task "packages : yum install packages" --inventory=/root/hosts --connection local /root/.ansible/pull/mgmt.*.internal/management.yml 
```

**Comment by christopheredsall** [comment]
It seems @milliams already solved this in commit 7dd04cb88c103f8511160db4fac0aaeffa498a65
